internal/serve/httphandler/disbursement_handler.go

```diff
@@ -170,21 +170,23 @@ func (d DisbursementHandler) GetDisbursements(w http.ResponseWriter, r *http.Req
 
 	ctx := r.Context()
 	disbursementManagementService := services.NewDisbursementManagementService(d.Models, d.DBConnectionPool, d.AuthManager)
-	resultWithTotal, err := disbursementManagementService.GetDisbursementsWithCount(ctx, queryParams)
+	resultWithTotal, err := disbursementManagementService.GetDisbursementsWithCount(ctx, queryParams, true)
```

Comment:
It seems that the only reason to add this boolean flag is so that you can simplify this test `Test_DisbursementManagementService_GetDisbursementsWithCount`. 

We should avoid changing the main code to accommodate tests, unless we're performing a refactor that fixes a design smell in the main code.

Reply:
it seems that inside the get disbursements handler is the only place where the orm method `GetDisbursementsWithCount` is used. my thinking was that if this could be re-used anywhere else and we didn't need the additional user metadata, we wouldn't want to run the additional logic of looping through potentially hundreds or thousands of disbursement records.

Reply:
If the concern is the performance of this call and we want to avoid making N+1 db queries, then the fix would be to : 
1. Collect the IDs of all users that need to be fetched. 
2. Perform one query to the users table to fetch users by IDs 
3. Re-map the users to the proper disbursements.